frontend/src/pages/statisticsPage/StatisticsPage.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, ButtonGroup } from "@mui/material";
+import { type FC, useState } from "react";
+import { SwitchButton } from "./StatisticsPage.style";
+import { TeamCardStatistics, PlayerCard, TeamCard } from "../../entity";
+import {
+  CustomDropdownMenu,
+  yearDropdownLabels,
+  filteringYears,
+} from "../../shared";
+import { playerMockData } from "../../entity/PlayerCard/model/playerMockData";
+
+const resp = ["data1", "data2", "data3", "data4", "data5"];
+
+const StatisticsPage: FC = () => {
+  const [teamsActive, setTeamsActive] = useState(true);
+  const [playersActive, setPlayersActive] = useState(false);
+
+  const onTeams = () => {
```

Comment:
```suggestion
  const onTeamsClick = () => {
```
имя функции должно быть с глаголом